icons/electronbuilder.svg

```diff
@@ -0,0 +1,4 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Before we can review this, please ensure it follows the description in the contributing guidelines. Most notably, t he logo should touch at least two edges of the viewbox (4 edge if the logo is square).